templates/baremetal-networks-routed.yaml

```diff
@@ -16,6 +16,11 @@ parameters:
       does not need to be changed.
     default: 172.17.0.0/24
 
+  overcloud_internal_net_allocation_pools:
+    type: json
+    description: Allocation pool for third overcloud_internal_net subnet
+    default: [{"start": "172.17.0.200", "end": "172.17.0.254"}]
```

Comment:
What we need is to make sure the OVB infrastructure allocation pool does not overlap with the one defined in the network_data.yaml or network-environment.yaml. We don't want the OVB infrastructure to allocate an IP for the router port in the range we use network-environment.yaml, we would end up duplicate ip's on OVB's router port and an overcloud node.

We would need the allocation pool in OVB to be large enough for the number of baremetal nodes + OVB infra ports such as the routers. I.e we could do 172.x.x.1-172.x.x.127 in OVB and 172.x.x.128-172.x.x.254 in network-environment.yaml ?

Maby using fixed ip's for the router's is better?
We fixed_ips the router at 172.x.x.254, and the either drop or match the allocation pool's (i.e 172.x.x.10-172.x.x.250)?

Reply:
Oh, bleh. Yeah, let's drop the allocation pool and just explicitly set the router IP. That's better than forcing the CIDR to be twice as large as it needs to be just to avoid overlap.